packages/legend-application-studio/src/components/editor/side-bar/CreateNewElementModal.tsx

```diff
@@ -347,10 +352,10 @@ const NewConnectionDriverEditor = observer(() => {
 
 const NewServiceDriverEditor = observer(() => {
   const editorStore = useEditorStore();
+  const runtimeSelectorPlaceholder = 'Choose a compatible runtime...';
```

Comment:
Instead of showing placeholder why can't we show the first compatible runtime option?